example/12_reduce/reduce_blockwise_impl.hpp

```diff
@@ -316,7 +316,16 @@ int reduce_blockwise_impl(bool do_verification,
 
     auto invoker_ptr = reduce.MakeInvokerPointer();
 
-    float avg_time = invoker_ptr->Run(argument_ptr.get(), StreamConfig{nullptr, time_kernel});
+    int log_level = 0, cold_niters = 5, nrepeat = 50;
+    // With beta != 0.0f there must be only one repeat for correct results since out memory is being
+    // overwritten
+    if(beta != 0.0f)
+    {
+        cold_niters = 0;
```

Comment:
Could you add some warning here? For example:
`std::cerr << "Warning: With beta != 0.0f there must be only one repeat for correct results since out memory is being overwritten."<< std::endl;`